pkg/test/deployment/helm.go

```diff
@@ -108,39 +108,38 @@ func HelmTemplate(deploymentName, namespace, chartDir, workDir, valuesFile strin
 		valuesFileString = fmt.Sprintf(" --values %s", valuesFile)
 	}
 
-	err := initCharts(chartDir, workDir)
-	if err != nil {
+	helmRepoDir := workDir + "/helmrepo"
```

Comment:
Please use/add paths in pkg/test/env

Reply:
workDir is actually a temp dir and helmRepo is created on top of it. This looks different from the other paths in istio/pkg/test/env/istio.go.

Reply:
nit: use `filepath.Join()` here and below.